web/src/components/ToFromDateFilters/ToFromDateFilters.tsx

```diff
@@ -41,120 +41,121 @@ export const ToFromDateFilters: FC<ToFromDateFilters> = ({
 );
 
 // Extra component to avoid the NDS filter group component passing invalid props (groupid/groupheading) into the div
-export const ToFromDateFiltersBody: FC<Except<ToFromDateFilters, "heading">> =
```

Comment:
weird linebreak/cf formatting is showing up as a diff

Reply:
I _think_ that's a prettier thing TBH - nothing I've done. I agree it's annoying it's made GH think the whole thing's changed. It hasn't!